report/mapping_functions.go

```diff
@@ -48,6 +48,37 @@ func ProcessPID(_ string, m NodeMetadata, grouped bool) (MappedNode, bool) {
 	}, show
 }
 
+// ProcessContainer maps Process topology nodes to the containers they run in.
+// We consider container and image IDs and names to be globally-scoped, and so
+// don't scope them further by e.g. host. If no container metadata is found,
+// nodes are grouped into the Uncontained node. If grouped is true, nodes are
+// grouped by container image.
```

Comment:
Yeah, its not quite that easy; container names are unique on a single machine, but not across a cluster.  The IDs should be unique (in that they are large and random), but there is no guarantee.  Images names and ids are enforced by the hub, so will be pretty unique, although you can make a local image with an arbitrary name, so I would only trust the IDs, which are content-addressable style hashes.